tests/winter_django/test_view.py

```diff
@@ -13,3 +22,25 @@ def test_create_django_urls():
 
     assert len(urls) == 1
     assert urls[0].pattern.regex.pattern == expected_url_pattern
+
+
+def test_create_django_urls_from_routes():
+    client = APIClient()
+    user = AuthorizedUser()
+    client.force_authenticate(user)
+    url = f'/notes/?note_id={uuid4()}'
+
+    get_http_response = client.get(url)
+    post_http_response = client.post('/notes/', data=dict(name='Name'))
+    patch_http_response = client.patch(url, data=dict(name='New Name'))
+
+    assert get_http_response.status_code == HTTPStatus.OK
+    assert post_http_response.status_code == HTTPStatus.OK
+    assert patch_http_response.status_code == HTTPStatus.OK
+
+
+def test_create_django_urls_from_routes_with_exception():
+    routes = find_package_routes('tests.api.exception_api')
```

Comment:
Maybe inline these routes in to the test? Otherwise I'm not why `find_package_routes('tests.api')` doesn't find it.